### PR TITLE
Fixed upstream component versions

### PR DESCRIPTION
**Pull request by vimalk78**
### Description
Fixed upstream component versions to
 - elasticsearch: 6.8.1
 - kibana: 6.8.1
 - fluentd: 1.7.4

/cc @igor-karpukhin @periklis 
/assign @jcantrill 

### Links

- JIRA: https://issues.redhat.com/browse/LOG-1700

### CODE REVIEW

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -21,7 +22,7 @@ LABEL \
         io.openshift.build.commit.id=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_COMMIT} \
         io.openshift.build.source-location=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_URL} \
         io.openshift.build.commit.url=${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_URL}/commit/${CI_ORIGIN_AGGREGATED_LOGGING_UPSTREAM_COMMIT} \
-        version=${CI_CONTAINER_VERSION}
+        version=v6.8.1
```
[the remark this answers is not in the document]
Comment (vimalk78) [reply elasticsearch/Dockerfile:25]: this label is used by OSBS to version the image

File: elasticsearch/Dockerfile [hunk 2]
```diff
@@ -3,9 +3,10 @@
 #@follow_tag(openshift-ose-base:ubi8)
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.7
 
-ENV BUILD_VERSION=${CI_CONTAINER_VERSION:-}
-ENV OS_GIT_MAJOR=${CI_X_VERSION:-}
-ENV OS_GIT_MINOR=${CI_Y_VERSION:-}
+ENV BUILD_VERSION=6.8.1
```
[the remark this answers is not in the document]
Comment (vimalk78) [reply elasticsearch/Dockerfile:6]: after some research i found that this variable is used to add OCP version in the Docker build.
so perhaps this variable can be deleted

**Review by periklis — CHANGES_REQUESTED** [review]
Missing the changes in curator image.

**Review by periklis — APPROVED** [review]
/lgtm

**Review by jcantrill — COMMENTED** [review]
/approve

**Pull request merged**